velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -1132,6 +1137,64 @@ TEST_F(DateTimeFunctionsTest, millisToTimestamp) {
       parseTimestamp("-292275055-05-16 16:47:04.192"));
 }
 
+TEST_F(DateTimeFunctionsTest, secondsToTimestamp) {
+  // Tests using integer seconds as input.
+  EXPECT_EQ(
```

Comment:
Also add test for int8_t to make registry worked, 
secondsToTimestamp<int8_t>(-1), parseTimestamp("1969-12-31 23:59:59"));
secondsToTimestamp<int16_t>(-1), parseTimestamp("1969-12-31 23:59:59"));
also for double and float

Reply:
Thanks, I pushed an update.